var/spack/repos/builtin/packages/attr/package.py

```diff
@@ -23,14 +28,14 @@ def url_for_version(self, version):
             url = "http://download.savannah.gnu.org/releases/attr/attr-{0}.src.tar.gz"
         return url.format(version)
 
+    # Ref. https://www.linuxfromscratch.org/blfs/view/7.5/postlfs/attr.html
     def configure_args(self):
         args = []
         args.append("--disable-static")
         return args
 
-    # Ref. https://www.linuxfromscratch.org/blfs/view/7.5/postlfs/attr.html
+    @when("@:2.4.47")
     def install(self, spec, prefix):
-        if self.version >= Version("2.4.48"):
-            make("install")
-        else:
-            make("install", "install-dev", "install-lib")
+        make("install", "install-dev", "install-lib")
+
+        make("install", "install-dev", "install-lib")
```

Comment:
@cosmicexplorer is this a typo? These looks to both be the same line to me.
```suggestion
        make("install", "install-dev", "install-lib")
```

Reply:
@cosmicexplorer ping

Reply:
ping @cosmicexplorer

Reply:
Yes! This was no longer necessary after some of the earlier fixes.